mock_gps_node:
  ros__parameters:
    sensor_gps: False  # Set to False for GPSINPUT
